tmt/steps/provision/local.py

```diff
@@ -71,6 +71,16 @@ def execute(self, command, **kwargs):
         # Run the command under the prepared environment
         return self.run(command, env=environment, shell=True, **kwargs)
 
+    def stop(self):
+        """ Stop the guest """
+
+        self.debug(f"Doing nothing to stop guest '{self.guest}'.")
+
+    def reboot(self, hard=False):
+        """ Reboot the guest, return True if successful """
+
+        self.debug(f"Doing nothing to reboot guest '{self.guest}'.")
```

Comment:
We should probably return `False` here as the `local` provision plugin just does not support reboots at all.

Reply:
Ok, I'll trust you.

Reply:
Fixed in 59016c27a30536f5b83a2fad345cfe62ecf47ce6